include/flatbuffers/flatbuffers.h

```diff
@@ -288,7 +288,7 @@ struct VectorIterator : public
   }
 
   VectorIterator operator++(int) {
-    VectorIterator temp(data_);
+    VectorIterator temp(data_,0);
     data_ += IndirectHelper<T>::element_stride;
```

Comment:
normally is good to keep bug fix commits separate from new features rather than mixing them in one commit.